consensus/reactor.go

```diff
@@ -357,7 +357,7 @@ func (conR *Reactor) ReceiveEnvelope(e p2p.Envelope) {
 				cs.Validators.Size(), cs.LastCommit.Size()
 			cs.mtx.RUnlock()
 
-			schema.WriteVote(conR.traceClient, height, round, msg.Vote, e.Src.ID(), schema.TransferTypeDownload)
+			schema.WriteVote(conR.traceClient, height, round, msg.Vote, e.Src.ID(), schema.Download)
```

Comment:
Do we need to save the height and round if we're saving the entire msg.Vote?

Reply:
unrelated to this PR, but I tend to agree. iirc this was to make it easier to find old or weird votes but was not necessary.

wdyt @staheri14? not in this PR, but does it make sense to remove the extra fields and combine the roundstate info we're collecting? we should have a complete view of when votes are sent and received, along with the round state of both nodes

Reply:
>Do we need to save the height and round if we're saving the entire msg.Vote?

The height and round of the vote do not always align with the receiver's height and round. Here, the `height` and `round` represent the receiving node's info, but not the vote.  

>wdyt @staheri14? not in this PR, but does it make sense to remove the extra fields and combine the roundstate info we're collecting? we should have a complete view of when votes are sent and received, along with the round state of both nodes

I support maintaining these extra field as it is simpler and more precise. If we don't retain this information, we would have to depend on the timestamps of the recorded data to establish correlations (except timestamps, is there any other way to correlate the two i.e., round state and recorded votes?).

Reply:
> The height and round of the vote do not always align with the receiver's height and round

That's a good point. Okay you've convinced me. Let's keep it.